canvas_modules/common-canvas/src/common-properties/controls/controls.scss

```diff
@@ -44,7 +44,8 @@
 		display: flex;
 		> div:first-child {
 			width: 100%;
-			overflow-x: auto;
+			min-width: 0; // Prevent overflow issues
+			flex: 1; // Allow the div to grow and shrink
```

Comment:
@srikant-ch5 Can you share an example of why `flex: 1` is needed?  Everything looks okay when I comment out this line.

Reply:
I have removed `flex` and updated the PR.